ztools/scada/opcua/helper.go

```diff
@@ -0,0 +1,96 @@
+package opcua
+
+import (
+	"encoding/hex"
+	"strconv"
+)
+
+
+func Checkerr(err error) error{
+	if err != nil {
```

Comment:
This is identical to the identify function / `return err`. I'm not sure why it's needed.